xbmc/filesystem/NFSFile.cpp

```diff
@@ -56,14 +66,21 @@ constexpr auto IDLE_TIMEOUT = 30s; // close fast unused contexts when no active
 constexpr int NFS4ERR_EXPIRED = -11; // client session expired due idle time greater than lease_time
 
 constexpr auto SETTING_NFS_VERSION = "nfs.version";
+constexpr auto SETTING_NFS_ENABLE_IDS = "nfs.enableids";
+constexpr auto SETTING_NFS_UID = "nfs.uid";
+constexpr auto SETTING_NFS_GID = "nfs.gid";
+constexpr auto SETTING_NFS_GROUP_PERMISSIONS = "nfs.grouppermissions";
+constexpr auto SETTING_NFS_OTHER_PERMISSIONS = "nfs.otherpermisions";
```

Comment:
```suggestion
constexpr auto SETTING_NFS_OTHER_PERMISSIONS = "nfs.otherpermissions";
```

Reply:
This reference is shared with xml settings.xml. So I'll commit the two files together.